src/ui/osx/TogglDesktop/Feature/TimeEntryEditor/DesktopLibraryBridge.m

```diff
@@ -192,15 +192,15 @@ - (void) registerUndoDeleteItem:(TimeEntryViewItem *)item
 
 - (void)updateDescriptionForTimeEntry:(TimeEntryViewItem *)timeEntry autocomplete:(AutocompleteItem *)autocomplete
 {
-	toggl_set_time_entry_project(ctx,
-								 [timeEntry.GUID UTF8String],
-								 autocomplete.TaskID,
-								 autocomplete.ProjectID,
-								 0);
-	toggl_set_time_entry_description(ctx,
-									 [timeEntry.GUID UTF8String],
-									 [autocomplete.Description UTF8String]);
-	[self updateTimeEntryWithTags:autocomplete.tags guid:timeEntry.GUID];
+    const char *tags = [[autocomplete.tags componentsJoinedByString:@"\t"] UTF8String];
+    toggl_set_time_entry(ctx,
+                         [timeEntry.GUID UTF8String],
+                         [autocomplete.Description UTF8String],
+                         autocomplete.TaskID,
+                         autocomplete.ProjectID,
+                         0,
+                         tags,
+                         timeEntry.billable);
```

Comment:
`billable` should come from `autocomplete` and not from `timeEntry`.
We need this to make sure autocompleting billable time entries sets the billable flag correctly. I believe the fact that we didn't do it before was a bug on macOS.